starport-core/src/main/scala/com/krux/starport/db/tool/SubmitPipeline.scala

```diff
@@ -37,6 +37,16 @@ object SubmitPipeline extends DateTimeFunctions with WaitForIt with DateTimeMapp
     pipelineDef.getField("MODULE$").get(null).asInstanceOf[DataPipelineDefGroup].schedule.asInstanceOf[RecurringSchedule]
   }
 
+  private def cleanupJar(jarFile: File, opts: SubmitPipelineOptions): Unit = {
+    if (opts.cleanUp) {
+      logger.info(s"Cleaning up JAR file...")
+      jarFile.deleteOnExit()
```

Comment:
let's add a comment here saying that in case of some uncaught exception, we have this to cleanup the file at JVM level